src/madsci_workcell_manager/madsci/workcell_manager/workcell_engine.py

```diff
@@ -618,6 +627,9 @@ def update_node(
         except Exception as e:
             error = Error.from_exception(e)
             node.status = NodeStatus(errored=True, errors=[error])
+            for error in node.status.errors:
+                if error.error_type == "ConnectionError":
+                    node.status.disconnected = True
```

Comment:
It might actually make sense to not filter by error type here, honestly, and mark the node as disconnected for any exception getting the status.